### PR TITLE
[22.05] backport #15446: Fix for new style conda packages 

### PR DESCRIPTION
**Pull request by bernt-matthias**
Backporting to 22.05 is required to get `planemo test` running again for tools with multi package requirements that do not have an image at quay.io yet, e.g. https://github.com/galaxyproject/tools-iuc/pull/5150 

## How to test the changes?
(Select all options that apply)
- [x] I've included appropriate [automated tests](https://docs.galaxyproject.org/en/latest/dev/writing_tests.html).
- [ ] This is a refactoring of components with existing test coverage.
- [ ] Instructions for manual testing are as follows:
  1. [add testing steps and prerequisites here if you didn't write automated tests covering all your changes]

## License
- [x] I agree to license these and all my past contributions to the core galaxy codebase under the [MIT license](https://opensource.org/licenses/MIT).

### CODE REVIEW

**Comment by mvdbeek** [comment]
> Backporting to 22.05 is required to get `planemo test` running again for tools with multi package requirements that do not have an image at quay.io yet, e.g. [galaxyproject/tools-iuc#5150](https://github.com/galaxyproject/tools-iuc/pull/5150)

I'm skeptical that this is the real reason it fails, this should only affect which base image we choose for building, but we should do this anyway.

**Comment by bernt-matthias** [comment]
See here: https://github.com/galaxyproject/tools-iuc/actions/runs/4263868544/jobs/7421208035#step:8:923

**Comment by mvdbeek** [comment]
Like I said before on another issue, that is not the reason the build fails

**Comment by bernt-matthias** [comment]
Thanks for the help. Not sure if it should be `conda_package_streaming` (as used in the org PR) or `conda-package-streaming` (which was the result of my stumbling around) :)

**Comment by mvdbeek** [comment]
https://pypi.org/project/conda-package-streaming/

File: pyproject.toml [hunk 1]
```diff
@@ -69,7 +70,6 @@ MarkupSafe = "*"
 Mercurial = "*"
 mrcfile = "*"
 nodeenv = "*"
-nose = "*"
```
Comment (nsoranzo) [inline pyproject.toml]: This and following dependency removals seem wrong?

Comment (mvdbeek) [reply pyproject.toml]: Yep, didn't mean to add that, but that was the only way to have poetry finish resolving.

**Pull request merged**